aspnetcore/fundamentals/w3c-logger/samples/6.x/Program.cs

```diff
@@ -0,0 +1,26 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Logging;
+
+namespace W3CLoggerSample
+{
+    public class Program
+    {
+        public static void Main(string[] args)
+        {
+            CreateHostBuilder(args).Build().Run();
+        }
+
+        public static IHostBuilder CreateHostBuilder(string[] args) =>
+            Host.CreateDefaultBuilder(args)
+                .ConfigureWebHostDefaults(webBuilder =>
+                {
+                    webBuilder.UseStartup<Startup>();
+                });
+    }
+}
```

Comment:
This is all 5.x code
6.x is coming out soon. We should create another issue to create a .NET 6 version.

Reply:
Can you change `aspnetcore/fundamentals/w3c-logger/samples/6.x/*` to `aspnetcore/fundamentals/w3c-logger/samples/5.x` ?

Reply:
W3C isn't available in 5.x, only 6.0+

Reply:
So why are you using Startup and other 5.x code and not the new minimal hosting model?

Reply:
Updated to use minimal hosting